sdks/go/test/regression/coders/fromyaml/fromyaml.go

```diff
@@ -51,6 +51,7 @@ var unimplementedCoders = map[string]bool{
 var filteredCases = []struct{ filter, reason string }{
 	{"logical", "BEAM-9615: Support logical types"},
 	{"30ea5a25-dcd8-4cdb-abeb-5332d15ab4b9", "https://github.com/apache/beam/issues/21206: Support encoding position."},
+	{"8c97b6c5-69e5-4733-907b-26cd8edae612", "https://github.com/apache/beam/issues/22629: Support single-precision float."},
```

Comment:
Noting there's a race with #22664. Whichever is merged later should remove this exception.

Reply:
@jrmccluskey this test case is still failing in Go. It looks like it may just be a test infra issue, but I'm not able to debug on my own. Could you take a look?

I'm re-opening https://github.com/apache/beam/issues/22629 to track

Reply:
Can you share the output from the test failure?

Reply:
Sorry I should have shared it to begin with but it had scrolled out of my shell history and I was lazy :grimacing: 

I dropped the log in https://github.com/apache/beam/issues/22629#issuecomment-1213398732